10_files_lab/fileslab.txt

```diff
@@ -1,37 +1,86 @@
 Files and Directories Lab
 Online: https://www.tocode.co.il/bundles/unix1/lessons/fileslab
 
-1. Create a new folder named "lab" in your home directory and inside create the files:
+1. 
+Create a new folder named "lab" in your home directory and inside create the files:
 
   - a.out, hero.c, enemy.c, game.c, main.c
   - human.h, monster.h
   - .highscore
 
-Write here the commands you used:
+Write here the commands you used:
+
+
+mkdir lab
+cd lab/
+touch  a.out
+touch  hero.c
+touch  enemy.c
+touch  game.c
+touch  main.c
+touch  human.h
+touch  monster.h
+touch  .highscore
+
 
 2. "cd" into lab folder and inside create the following subdirectories:
   - Music
   - Misc
   - Drivers
 
-Write here the commands you used:
+Write here the commands you used:
+
+
+
+mkdor Musidc Misc Drivers
+
+
 
 3. Copy the file "/etc/passwd" to each subdirectory created in (2).
 
-Write here the commands you used:
+Write here the commands you used:
+
+
+
+cp -r /etc/passwd Drivers/
+cp -r /etc/passwd Misc/
+cp -r /etc/passwd Music/
+

```

Comment:
No need for `-r` here. A simple `cp /etc/passwd Drivers` works well. Use `-r` only when copying directories.